src/backend/gpopt/gpdbwrappers.cpp

```diff
@@ -2736,47 +2739,94 @@ gpdb::IsTypeRange(Oid typid)
 	return false;
 }
 
-RowMarkClause *
-gpdb::GetParseRowmark(Query *query, Index rtindex)
```

Comment:
We could resolve conflicts without removing the previous code and just change its location inside a file. I guess it is legit to make such changes for **our** patches.  We **do not need** a separate commit 2809cad07593997680149fdffca9fb3f1c89b2b8.